stylesheet.css

```diff
@@ -0,0 +1,56 @@
+.home-page--container {
+    align-items: center;   
+    background-color: #202020;
+    display: flex;
+    font-family: 'Lato', sans-serif;
+    height: 100vh;
+    justify-content: center;
+}
+
+.home-page--profile-info-container {
+    align-items: center;
+    background-color: #181818;
+    box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.15);
+    display: flex;
+    flex-direction: column;
+    width: 260px;
+}
+
+.home-page--profile-info-container:not(:last-child) {
+    margin-right: 24px;
+}
+
+.home-page--profile-img {
+    border-radius: 2px;
```

Comment:
the image can't have border-radius in the bottom.  You should use something like this:
```
border-radius: 2px 2px 0 0;
```